build/common.nix

```
@@ -17,61 +14,71 @@ with lib;
 
   nixpkgs.config.allowUnfree = true;
 
-  services.openssh.authorizedKeysFiles = mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
-
-  boot.kernelModules = [ "coretemp" ];
-
-  # Prevent "out of sync" errors on the KVM switch.
-  boot.vesa = false;
-  boot.blacklistedKernelModules = [
-    "radeonfb"
-    "radeon"
-    "i915"
-  ];
-  boot.kernelParams = [ "nomodeset" ];
-
   hardware.enableAllFirmware = true;
   hardware.cpu.amd.updateMicrocode = true;
   hardware.cpu.intel.updateMicrocode = true;
 
-  environment.systemPackages = [
-    pkgs.emacs
-    pkgs.sysstat
-    pkgs.hdparm
-    pkgs.sdparm # pkgs.lsiutil
-    pkgs.htop
-    pkgs.sqlite
-    pkgs.iotop
-    pkgs.lm_sensors
-    pkgs.hwloc
-    pkgs.lsof
-    pkgs.numactl
-    pkgs.gcc
-    pkgs.smartmontools
-    pkgs.tcpdump
-    pkgs.gdb
-    pkgs.elfutils
-  ];
+  boot.kernel.sysctl = {
+    # reboot on kernel panic
+    "kernel.panic" = 60;
+    "kernel.panic_on_oops" = 1;
+  };
 
-  services.openssh.enable = true;
+  documentation.nixos.enable = false;
 
-  boot.kernel.sysctl."kernel.panic" = 60;
-  boot.kernel.sysctl."kernel.panic_on_oops" = 1;
+  environment = {
+    enableDebugInfo = true;
+    systemPackages = with pkgs; [
+      # debugging
+      gdb
+      lsof
+      sqlite
```

```suggestion
      sqlite-interactive
```
otherwise the cli is not usable and I guess this is what we want here